distribution/lib/Standard/Table/0.0.0-dev/src/Internal/Parse_To_Table.enso

```diff
@@ -15,10 +15,12 @@ from project.Errors import Duplicate_Output_Column_Names
    Converts a Text into a Table using a regular expression pattern.
 
    See Table.parse_text_to_table.
-parse_text_to_table : Text -> Text -> Case_Sensitivity -> Boolean -> Problem_Behavior -> Table ! Type_Error | Regex_Syntax_Error | Illegal_Argument
-parse_text_to_table text pattern_string="." case_sensitivity=Case_Sensitivity.Sensitive parse_values=True on_problems=Report_Warning =
+parse_text_to_table : Text | Regex -> Text -> Case_Sensitivity -> Boolean -> Problem_Behavior -> Table ! Type_Error | Regex_Syntax_Error | Illegal_Argument
+parse_text_to_table text regex_or_pattern_string="." case_sensitivity=Case_Sensitivity.Sensitive parse_values=True on_problems=Report_Warning =
     case_insensitive = case_sensitivity.is_case_insensitive_in_memory
-    pattern = Regex.compile pattern_string case_insensitive=case_insensitive
+    pattern = case regex_or_pattern_string of
+        _ : Regex -> regex_or_pattern_string
```

Comment:
```suggestion
        _ : Regex -> regex_or_pattern_string.recompile case_sensitivity
```

Will need to be rebased on my one for this to work.

Reply:
Done.